custom-actions/media-panel/create_dated_folder.py

```diff
@@ -0,0 +1,34 @@
+#Creates a Dated Folder
+def get_media_panel_custom_ui_actions():
+
+    def scope_libraries(selection):
+        import flame
+        for item in selection:
+            if isinstance(item, (flame.PyLibrary)):
+                return True
+        return False
+
+    def dated_folder(selection):
+        import flame
+        import os
```

Comment:
The generic import should be moved to the top of the file.
The flame import needs to be there however since the script might be loaded before it is available.

Reply:
Actually, I'm not sure the flame & os import are needed and could be removed.